### PR TITLE
Update README and remove credstash where still in use.

### PR DESCRIPTION
**Pull request by jschwartzentruber**

### CODE REVIEW

**Review by pyoor — APPROVED** [review]
Great work!  Looks good to me.

File: services/libfuzzer/README.md [hunk 1]
```diff
@@ -26,12 +50,13 @@ It is recommended to reserve at least 4GB of memory for containers running cover
 ### Example: LibFuzzer Run
 
 ```bash
-fuzzfetch --fuzzing --coverage -a --gtest -n firefox
+fuzzfetch --fuzzing --coverage --gtest -n firefox
```
Comment (pyoor) [inline services/libfuzzer/README.md:53]: Remove `--coverage`

Comment (pyoor) [reply services/libfuzzer/README.md:53]: Also, there doesn't appear to be an fuzzing-opt so maybe use `-a --fuzzing --gtest`

File: services/libfuzzer/README.md [hunk 2]
```diff
@@ -1,17 +1,41 @@
...
+
+... add any environment variables required by the fuzzer using `-e VAR=value`
 
 ### Example: LibFuzzer Coverage Run
 
 ```bash
-REVISION=$(curl -sL https://build.fuzzing.mozilla.org/builds/coverage-revision.txt)
-fuzzfetch --build "$REVISION" --fuzzing --coverage -a --gtest -n firefox
+REVISION="$(curl --compressed -sSL https://community-tc.services.mozilla.com/api/index/v1/task/project.fuzzing.coverage-revision.latest/artifacts/public/coverage-revision.txt)"
+fuzzfetch --build "$REVISION" --fuzzing --coverage --gtest -n firefox
```
Comment (pyoor) [inline services/libfuzzer/README.md:32]: The gtest artifact does not exists for this build: https://firefox-ci-tc.services.mozilla.com/tasks/index/gecko.v2.mozilla-central.latest.firefox/linux64-ccov-fuzzing-opt

File: services/libfuzzer/README.md [hunk 3]
```diff
@@ -1,17 +1,41 @@
...
 
 docker run \
     -h `hostname` \
     -v ~/.fuzzmanagerconf:/home/worker/.fuzzmanagerconf \
     -v ~/firefox/:/home/worker/firefox \
+    -e NO_SECRETS=1 \
```
Comment (pyoor) [inline services/libfuzzer/README.md:38]: You'll also have to include:
```
export GCOV_PREFIX_STRIP=4
export GCOV_PREFIX=/home/worker/firefox
```

Comment (jschwartzentruber) [reply services/libfuzzer/README.md:38]: These are already set [in coverage.sh](https://github.com/MozillaSecurity/orion/blob/4b2b8998ebeadb4be43006174fae8373430d4e10/services/libfuzzer/coverage.sh#L42-L50)

**Comment by jschwartzentruber** [comment]
CI fails because of #128 ... merging anyways.

**Pull request merged**